gtfu/src/main/java/gtfu/TripCollection.java

```diff
@@ -80,6 +80,9 @@ public TripCollection(String path, StopCollection stopCollection, ShapeCollectio
                 if (Util.isEmpty(time)) continue;
 
                 trip = map.get(tripID);
+
+                if (trip == null) continue;
```

Comment:
Fail message was sent in above lines, this "continue" is so we can work with the trips that DON'T lack a shape

Reply:
Note that this bug prevented missing-route errors for showing in addition to missing-shape errors.